### DISCUSSION TITLE
Hello, I would like to know how to use xterm.js in Nuxt3

### DISCUSSION THREAD

**Discussion by Thinker-feiluo** [sourcelair/xterm.js#5047] [Q&A]
Hello, I would like to know how to use xterm.js in Nuxt3

**Comment by jerch** [comment]
Since you got no answer so far - your best bet prolly is to learn about your framework's life-cycling of components and do the integration yourself.

Regarding xterm.js it is important, that the terminal instance and the terminal DOM nodes should survive re-rendering cycles, so ideally you find a way to leave them untouched by your framework. If you know some react - there are examples floating around showing that for react.
